Contributors.txt

```diff
@@ -108,25 +109,29 @@ Developers:
 	Lynn Root
 	Pete Rowley
 	Lenka Ryznarova
+	Alexander Scheel
 	Thorsten Scherf
 	shanyin
 	Kaleemullah Siddiqui
 	Michael Simacek
 	Lars Sjostrom
 	Filip Skola
 	Aleksei Slaikovskii
+	Alexey Slaykovsky
```

Comment:
Should we ask him which name he prefers?

Reply:
@slaykovsky which name do you prefer to use in the Contributors list?

Reply:
Ok, let me just drop the new addition for @slaykovsky  and keep using the old one right now.